config/webpacker.yml

```diff
@@ -7,7 +7,7 @@ default: &default
   public_output_path: packs
   cache_path: tmp/cache/webpacker
   check_yarn_integrity: false
-  webpack_compile_output: false
+  webpack_compile_output: true
```

Comment:
We will want to test this setting out again since the reason it was disabled before now eludes me. Likely was related to slow startup which might not be an issue now..

Reply:
@thestephenmarshall if it helps, I set this to true to see CI compilation errors, there's a thread about it (https://github.com/rails/webpacker/issues/955) and they've decided to set it to `true` by default on a next version https://github.com/rails/webpacker/pull/2316

Reply:
Okay then if that is the case let's set it to true as well.